### ISSUE TITLE
[BUG] ./install failed on master branch and dev/0.7.0 branch

### ISSUE THREAD

**Issue by ellieeee** [CoinAlpha/hummingbot#250]
**Describe the bug**
./install failed on master branch and dev/0.7.0 branch

**Steps To Reproduce**
1. git clone https://github.com/CoinAlpha/hummingbot.git
2. cd hummingbot
3. ./install

**Screenshots**
```
~/echio/hummingbot$ ./install
Using Anaconda API: https://api.anaconda.org
Fetching package metadata .........

NoPackagesFoundError: Package missing in current linux-64 channels:
  - asn1crypto 0.24.0 py36_0

martin_kou@Valhalla:~/echio/hummingbot$ git branch
* master
```

**Release version**
master
dev/0.7.0

**Attachments**
// Please attach your config file and log file to help us debug. It would be difficult for us to help you without those! 

*WARNING: Do NOT publish any exchange API keys or your wallet's private key. Whoever has access to them may steal your assets!*

**Optional: your discord username: ellieeeee#4149**

**Comment by ellieeee** [comment]
```
$ git clone https://github.com/CoinAlpha/hummingbot.git
Cloning into 'hummingbot'...
remote: Enumerating objects: 4, done.
remote: Counting objects: 100% (4/4), done.
remote: Compressing objects: 100% (4/4), done.
remote: Total 4744 (delta 0), reused 1 (delta 0), pack-reused 4740
Receiving objects: 100% (4744/4744), 7.65 MiB | 0 bytes/s, done.
Resolving deltas: 100% (3349/3349), done.
Checking connectivity... done.
$:~/echio$ cd hummingbot/
$:~/echio/hummingbot$ ls
bin                 compile      CONTRIBUTING.md  documentation  jenkins  README.md  test
clean               compile.bat  Dockerfile       hummingbot     LICENSE  setup      uninstall
CODE_OF_CONDUCT.md  conf         DOCKER.md        install        logs     setup.py   wings
$:~/echio/hummingbot$ ./install
Using Anaconda API: https://api.anaconda.org
Fetching package metadata .........

NoPackagesFoundError: Package missing in current linux-64 channels:
  - asn1crypto 0.24.0 py36_0

$:~/echio/hummingbot$ git branch
* master
```

**Comment by ellieeee** [comment]
Updating the Anaconda to the latest version solves the problem, and can clearly see the package is available in 0.24.0=py3.6 version. This version of conda worked earlier though, so not too sure why it happened.

`$ conda update --prefix <anaconda installation path> anaconda`
